onnxruntime/test/onnx/microbenchmark/modeltest.cc

```diff
@@ -16,3 +18,43 @@ static void BM_LoadModel(benchmark::State& state) {
 }
 
 BENCHMARK(BM_LoadModel);
+
+extern OrtEnv* env;
+
+#define ORT_BREAK_ON_ERROR(expr)                         \
+  do {                                                   \
+    OrtStatus* onnx_status = (expr);                     \
+    if (onnx_status != NULL) {                           \
+      state.SkipWithError(OrtGetErrorMessage(onnx_status)); \
+      OrtReleaseStatus(onnx_status);                     \
+    }                                                    \
+  } while (0);
+
+#ifdef USE_CUDA
+static void BM_CreateSession_WithGPU(benchmark::State& state) {
+  const char* model_path = "../models/opset8/test_bvlc_alexnet/model.onnx";
+  OrtSessionOptions* session_option = OrtCreateSessionOptions();
```

Comment:
Do session_options requires releasing?

Reply:
Hi @jignparm , I fixed it, could you please review it again?